packages/vhd-lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vhd-lib",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

Comment:
@pdonias What do you think about incrementing the version directly in a feature PR in comparison to do it during the release process?

Reply:
What if there are more than 1 PR between 2 releases?

Reply:
IMO it's not a good idea to have package versions in `master` that are not published yet.

@nraynaud please remove these changes :slightly_smiling_face: